### DISCUSSION TITLE
Align Start Menu to Left Hand Corner Of Screen

### DISCUSSION THREAD

**Discussion by philhemel** [Open-Shell/Open-Shell-Menu#1287] [General]
Hi, is there any way to force the Start Menu to open on the left hand corner of the screen (for Windows 11)
I'm happy with the Start Button being in the middle, but would like the MENU to open far left
I've tried "Align start menu to working area" but that doesn't help
"Horizontal Position Offset: 1000" proves it's easy for the start menu to open away from the start button, but -1000 doesn't work to move it the other way!! (and wouldn't be a reliable way of aligning to the left hand corner)
Many thanks x

**Comment by KrisMue** [comment]
I have similar problem. I would like to have the open shell button on the left and the original windows 11 start button centered. This was the default with older versions of open shell. Neither "Start Button -> Custom -> Align to corner of the screen" nor a negative vertical offset did the trick. Any ideas? Thanks.

**Reply by antonderuiter** [reply]
This option is missing in Open Shell, but possible in Classic Shell, and reason for me not to switch to Open Shell.

**Comment by f14tomcat2** [comment]
The functionalities of the Open Shell start button and the Windows start button sometimes clash making the overlay a dual function problematic.  Example......  if hover is chosen as the option to open the Open Shell menu, right click the start button to get the Windows options makes you quickly move the cursor up into the Windows Menu area or the Open Shell menu will open overlaying the Right Click menu.  As I posted before in another discussion here, Open Shell menu and Windows menu are functionally far apart and need the option to remain separate, i.e. Open Shell button on left corner and Windows button centered (floating).

**Comment by jen0ch** [comment]
Hello! Found the solution on reddit, just in case you or anyone else was still looking. You have to change taskbar behaviour in the control panel - search for taskbar behaviour and check "align to left". https://www.reddit.com/r/OpenShell/comments/11aif2a/possibly_to_left_align_the_menu/

**Comment by peterjay733** [comment]
No, that doesn't actually solve the "problem." "We" would like the taskbar items centered, but the Open Shell start button to be on far-left. This is the "old" behavior in Open Shell. That changed, sadly. Looking for someone who can revert this in the current Open Shell version. Will entertain the notion of paying a reasonable amount for this.

**Reply by HelloGames-ds** [reply]
I have the same problem, now I'm trying to do something using windhawk additionally, but so far without success. Maybe there is a solution in windhawk (at least move the start menu icon to the left), but I did not find it.

**Comment by amymor** [comment]
There is an option for it in StartAllBack:
<img width="1404" height="839" alt="image" src="https://github.com/user-attachments/assets/11c91de3-08c5-4129-bbb6-7b185bfd356d" />
